tests/readme_examples.cpp

```diff
@@ -305,6 +305,30 @@ bool is_correct_string() {
   return is_ok;
 }
 
+void parse_documentation() {
+  const char *json      = "{\"key\":\"value\"}";
+  const size_t json_len = strlen(json);
+  simdjson::dom::parser parser;
+  // We comment out element since it is unused.
+  //simdjson::dom::element element = 
+  parser.parse(json, json_len);
+}
+
+
+void parse_documentation_lowlevel() {
+  // such low-level code is not recommended
+  // Motivation: https://github.com/simdjson/simdjson/issues/1175
+  const char *json      = "{\"key\":\"value\"}";
+  const size_t json_len = strlen(json);
+  std::unique_ptr<char[]> padded_json_copy{new char[json_len + SIMDJSON_PADDING]};
+  memcpy(padded_json_copy.get(), json, json_len);
+  memset(padded_json_copy.get() + json_len, 0, SIMDJSON_PADDING);
+  simdjson::dom::parser parser;
+  // We comment out element since it is unused.
+  //simdjson::dom::element element = 
```

Comment:
Could this be removed?

Reply:
Same remark. I fixed this by uncommenting but adding a line to avoid the unused warning.

We do want to mimic our example code as closely as possible (hygiene).